src/test/java/com/google/devtools/build/lib/worker/WorkerFactoryTest.java

```diff
@@ -48,7 +48,8 @@ public void sandboxedWorkerPathEndsWithWorkspaceName() throws Exception {
             "dummy",
             HashCode.fromInt(0),
             ImmutableSortedMap.of(),
-            true);
+            true,
+            false);
```

Comment:
This is unreadable, isn't it? May be worth turning to an enum instead of a boolean.

Reply:
This interface is defined in WorkerKey. Do you mean changing the constructor of WorkerKey or just making this test more readable?

Reply:
You can just add `/*paramName=*/false` to the code to disambiguate these. I just learned a few days ago that the contents of these magic comments are programmatically validated by error-prone, so they are good enough given this existing interface.

Reply:
Done.